packages/server/tests/acceptance.spec.ts

```diff
@@ -209,6 +214,68 @@ describe('RPC Server Integration Tests', async function () {
         expect(blockResult.transactions.length).to.be.greaterThan(0);
     });
 
+    it('should execute "eth_getBalance" for primary account', async function () {
+        const res = await callSupportedRelayMethod(this.relayClient, 'eth_getBalance', [mirrorPrimaryAccount.evm_address, 'latest']);
+        expect(res.data.result).to.not.be.equal('0x0');
+        expect(res.data.result).to.eq('0x566527b339630a400');
```

Comment:
Why do we have 2 separate `expect`s for the same value. If we are doing `expect(res.data.result).to.eq('0x566527b339630a400');` do we need `expect(res.data.result).to.not.be.equal('0x0');`?